### ISSUE TITLE
Fail to install without admin privileges in Windows

### ISSUE THREAD

**Issue by pauloneves** [gitextensions/gitextensions#3888]
**Do you want to request a *feature* or report a *bug*?**

bug

**What is the current behavior?**

Fails to install if user doesn't have admin privileges. 

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

I want to install just for my user. The initial screen says that I don't need admin privileges:
![image](https://user-images.githubusercontent.com/221084/29032139-c964dcf2-7b66-11e7-959a-cdab55c4fa6b.png)

I choose another install location, since I don't have write access to the default one. 

I select the default options and Putty.

Then I get the following error:
![image](https://user-images.githubusercontent.com/221084/29032238-1b6cf9d0-7b67-11e7-874f-7c6bea184851.png)

**What is the expected behavior?**

Expected a successful instalation. 

**Environment you encounter the issue:**
Windows 7 professional 64 bits 

- GitExtensions version: 2.50.1
- GIT version: 
- OS version: 

**Did this work in previous version of GitExtensions (which)?**

**Comment by aryoda** [comment]
What is the current status regarding "installation without admin privileges"? Does it work now?

**Comment by vbjay** [comment]
Try the rc1.  Let us know.  If you have GE installed you may need to
download that installer also to have the installer remove the old version.

So if you have 2.51.05 installed, download the 2.51.05 installer from
https://github.com/gitextensions/gitextensions/releases

**Comment by RussKie** [comment]
Probably the same.
I don't believe anyone really gave any effort to this issue.

Until this is fixed you can use the portable version

**Comment by aryoda** [comment]
Yes, the portable version is workaround (though I still had a "crash" when comparing different file versions - I am trying to make this reproducible...)

**Comment by VoltanFr** [comment]
Hello
For what it's worth, just wanted to mention that we're in the same boat. Our company is putting high pressure on removing admin rights for security reasons.
The portable version is a big service drop because the contextual menus in explorer is important.
It sounds like the registration of the explorer integration could be done in HKCU instead of HKLM?
Is there any work in progress on this subject? Or should I consider creating a PR?
Thanks

**Comment by gerhardol** [comment]
> It sounds like the registration of the explorer integration could be done in HKCU instead of HKLM? Is there any work in progress on this subject? Or should I consider creating a PR? Thanks

No work is ongoing what I know, PRs are welcome.
You may want to state what to change first (in this or a new issue) or if the change is easy enough to create a WIP PR.

**Comment by aryoda** [comment]
Perhaps this SO post provides the required knowledge to implement that in HKCU:

https://stackoverflow.com/questions/1387769/create-registry-entry-to-associate-file-extension-with-application-in-c

**Comment by SuperJugy** [comment]
I still have this problem with the latest version of GitExtensions in Windows 10. Is there a fix?

**Comment by mstv** [comment]
With #10421, you should not need an installer.
You could try the recent portable build https://ci.appveyor.com/api/buildjobs/lwvjb88lj0qr0eb0/artifacts/artifacts%2FRelease%2Fpublish%2FGitExtensions-Portable-4.1.0.16207-e35d1445e.zip
Does it work for you?

**Comment by SuperJugy** [comment]
Is the portable version updatable? last time I used the portable version I had issues updating it. I had to get a new portable version and painstakingly copy each configuration by hand again. If the portable version is updatable, then yes, this helps. if not, I prefer the ability to install without admin rights just for the local user.

**Comment by mstv** [comment]
The portable version can be updated. Just replace the files if it is not a major update.
For major updates delete all files and folders except:
* file `GitExtensions.settings`
* file `WindowPositions.xml`
* entire folder `Recent`

Or else change the "portable" setting in `GitExtensions.dll.config`.
Then your settings will be stored in your user profile.
```
  <applicationSettings>
    <GitCommands.Properties.Settings>
      <!--Set this setting to True to store settings with the application, False to  store settings in user's application data path.-->
      <setting name="IsPortable" serializeAs="String">
        <value>True</value>
      </setting>
    </GitCommands.Properties.Settings>
  </applicationSettings>
</configuration>
```

**Comment by SuperJugy** [comment]
This is nice to know. Thanks a lot.
